apyros/metalog.py

```diff
@@ -57,7 +71,11 @@ def getLog( self, prefix ):
                 ret = line.split()[1].strip()
```

Comment:
What you had fails in different strange cases like space in the file path. In general, I try to be as explicit as possible and I don't spare asserts for places I know are not supported. So if space or colon cannot be part of filename or prefix, go ahead and just assert so. That is the best documentation I know of.